src/runtime_src/core/edge/user/aie/common_layer/adf_runtime_api.h

```diff
@@ -23,6 +23,7 @@
 #include <queue>
 #include <vector>
 
+#ifdef XRT_ENABLE_AIE
```

Comment:
This whole file is under XRT_ENABLE_AIE flag. Can you please update CMAKe to reflect this instead of whole file under ifdef.